tests/pdctl/helper.go

```diff
@@ -33,6 +33,15 @@ import (
 	"github.com/tikv/pd/tests"
 )
 
+// StoreState_value is used for store state in PD definition
+var StoreState_value = map[string]int32{
+	"Up":           0,
+	"Disconnected": 0,
```

Comment:
`Disconnected` could be `Offline` at the same time.

Reply:
has romoved it